backend/social_connections/tests/test_discord_oauth.py

```
@@ -40,6 +41,27 @@ def setUp(self):
         )
         self.service = DiscordOAuthService()
 
+    def attach_session(self, request):
+        middleware = SessionMiddleware(lambda req: None)
```

Ruff (0.15.20)

[warning] Unused lambda argument: `req`

(ARG005)